module/account/src/Application/Command/CreateUserCommand.php

```diff
@@ -97,6 +97,6 @@ public function execute(InputInterface $input, OutputInterface $output): int
             $output->writeln(sprintf('<error>Can\'t find role %s</error>', $role));
         }
 
-        return 1;
+        return 0;
```

Comment:
Shouldn't this return code conditionally? The above line seems like an error occurred.